aws/ecs/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "cs-aws-infrastructure",
+  "version": "0.1.0",
+  "bin": {
+    "ckeditor-cs-on-premise-infrastructure": "bin/ckeditor-cs-on-premise-infrastructure.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk"
+  },
+  "devDependencies": {
+    "@aws-cdk/assert": "1.76.0",
+    "@cksource-cs/eslint-config-cs-module": "^2.1.1",
+    "@types/jest": "^26.0.19",
+    "@types/node": "10.17.27",
+    "aws-cdk": "1.76.0",
```

Comment:
This is autogenerated and probably could be removed. But I'd prefer to keep it and use it instead of global dependencies. I like it this way because it's a more stable approach.